src/BrazeKit-dev.js

```diff
@@ -22,11 +22,6 @@ var name = 'Appboy',
         PageView: 3,
         PageEvent: 4,
         Commerce: 16,
-    },
-    CommerceEventTypes = {
```

Comment:
I think you can get away with making the following assignment, which might result in less changes in this PR but might set us up to destructure later.

```
const CommerceEventTypes = mParticle.CommerceEventType;
```

That way in later lines, you can continue to use `CommerceEventTypes.ProductAddToWishlist`.